static/src/stylesheets/module/_adslot.scss

```diff
@@ -69,9 +69,17 @@
 }
 
 .sticky-top-banner-ad {
+    transform: translate3d(0, 0, 0);
+    will-change: transform;
```

Comment:
Come to think of it, moving it inside the animation class is a good proxy. Since adding the class will happen before and changes in height that should be animated.